### ISSUE TITLE
Lisää "korjaa näin" -ohje jokaisen virheen yhteyteen

### ISSUE THREAD

**Issue by Kaaponen** [Kaaponen/a11y-python-monitor#18]
Laajenna raportteja niin, että:
	•	Jokaisen WCAG/axe-virheen yhteyteen tulee lyhyt ohje tai esimerkki
	•	Esim: “Contrast insufficient” → “Käytä tummempaa tekstiä tai vaaleampaa taustaa”
